### PR TITLE
usage of tldextract is responsible of a significative performance decrease

### PR DESCRIPTION
**Pull request by kalessin**
@sibiryakov  as you suggested I have profiled jobs and process spents an important amount of time (near 30-40% of total) in cPickle.load() method, which is used by tldextract. Disabling DomainMiddleware jobs works much faster, and are not needed by any CF component. So this middleware should be enabled only when required.

### CODE REVIEW

File: crawlfrontier/settings/default_settings.py [hunk 1]
```diff
@@ -6,9 +6,9 @@
 REQUEST_MODEL = 'crawlfrontier.core.models.Request'
 RESPONSE_MODEL = 'crawlfrontier.core.models.Response'
 MIDDLEWARES = [
-    'crawlfrontier.contrib.middlewares.domain.DomainMiddleware',
+    # 'crawlfrontier.contrib.middlewares.domain.DomainMiddleware',
```
Comment (sibiryakov) [inline crawlfrontier/settings/default_settings.py]: IMO, it doesn't make sense to leave comments there. One could easily find them in docs or in sources. So please feel free to remove them.

Comment (kalessin) [inline crawlfrontier/settings/default_settings.py]: ok. wasn't sure to remove, but I like more to remove :)

**Comment by sibiryakov** [comment]
It would be great to have a tests checking that all components in request.meta that are required for basic CF functionality are added by middlewares. So removing some important middleware or braking it would cause test to fail.

**Comment by sibiryakov** [comment]
Please also update the docs for frontier settings.

**Comment by sibiryakov** [comment]
Overall finding is great!

**Pull request merged**